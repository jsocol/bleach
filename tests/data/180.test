<div id="112"><div style="font-family:foo{bar;background:url(http://foo.f/oo};color:red/*/foo.jpg);">X</div>//["'`-->]]>]</div><div id="113"><div id="x">XXX</div>
<style>

#x{font-family:foo[bar;color:green;}

#y];color:red;{}

</style>//["'`-->]]>]</div>
