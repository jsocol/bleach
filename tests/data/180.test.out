<div id="112"><div style="font-family:foo{bar;background:url(http://foo.f/oo};color:red/*/foo.jpg);">X</div>//["'`--&gt;]]&gt;]</div><div id="113"><div id="x">XXX</div>
<style>

#x{font-family:foo[bar;color:green;}

#y];color:red;{}

</style>//["'`--&gt;]]&gt;]</div>
